src/screens/ParadeMapScreen/Map.js

```diff
@@ -132,21 +131,22 @@ class Map extends Component<Props, State> {
           ))}
         </MapView>
         {!neverAskStatuses.has(this.state.locationPermission) && (
-          <Touchable
-            onPress={this.moveToCurrentLocation}
-            style={styles.touchable}
-            background={TouchableNativeFeedback.SelectableBackgroundBorderless()}
-          >
-            <Image
-              accessibilityLabel="Show my location"
-              source={
-                this.state.atUserLocation
-                  ? locationButtonActive
-                  : locationButtonInactive
-              }
-              style={styles.image}
-            />
-          </Touchable>
+          <View style={styles.touchable}>
```

Comment:
Do we need this additional view?

Reply:
yes, because TouchableWithoutFeedback doesn't support the style prop and so is displaying on the wrong side of the screen without the wrapping view. tbf I should remove that null style from the touchable in that case...

Reply:
Ah right. Yeah, if you could remove the `style` from the touchable, that would be great. I really wondered how this still looks good 🙂